subsys/net/lib/nrf_provisioning/src/nrf_provisioning_coap.c

```diff
@@ -482,17 +484,18 @@ static int request_commands(struct coap_client *client,
 {
 	int ret;
 	char after[NRF_PROVISIONING_CORRELATION_ID_SIZE];
-	char *rx_buf_sz = STRINGIFY(CONFIG_NRF_PROVISIONING_RX_BUF_SZ);
-	char *tx_buf_sz = STRINGIFY(CONFIG_NRF_PROVISIONING_TX_BUF_SZ);
+	const char *rx_buf_sz = STRINGIFY(CONFIG_NRF_PROVISIONING_RX_BUF_SZ);
+	const char *tx_buf_sz = STRINGIFY(CONFIG_NRF_PROVISIONING_TX_BUF_SZ);
+	const char *limit = STRINGIFY(CONFIG_NRF_PROVISIONING_CBOR_RECORDS);
```

Comment:
I just realized:
```suggestion
	const char rx_buf_sz[] = STRINGIFY(CONFIG_NRF_PROVISIONING_RX_BUF_SZ);
	const char tx_buf_sz []= STRINGIFY(CONFIG_NRF_PROVISIONING_TX_BUF_SZ);
	const char limit[] = STRINGIFY(CONFIG_NRF_PROVISIONING_CBOR_RECORDS);
```
would allow `sizeof(limit) -1` to give same answer as `strlen(limit)`.

However, this is irrelevant. Compiler probably optimizes this anyway.